tests/src/test/java/alluxio/server/ft/journal/TableMasterJournalIntegrationTest.java

```diff
@@ -49,9 +51,14 @@
  * Integration tests for table master functionality.
  */
 public class TableMasterJournalIntegrationTest {
+  private static final int WORKER_CAPACITY_BYTES = 200 * Constants.MB;
```

Comment:
Isn't this quite a large amount?

Reply:
transformation has a default blocksize that is quite large. This is to accommodate that.

Reply:
Can we reduce the default block size? 200MB is a lot to require for a small integration test...

Reply:
what is the negative effect of this? 
The size is determined by ParquetWriter minimum file size which is 128 MB. This is not the alluxio block size.